Tests/smokeTest.js

```diff
@@ -0,0 +1,28 @@
+var homepage = require('../Pages/homePage');
```

Comment:
test name should be more descriptive, not "smoke test"

Reply:
For now, we arranged that this title will remain